src/components/Search/OmniBox.js

```diff
@@ -78,6 +80,10 @@ const OmniBox = ({ isHomePage = false, t }) => {
     doSearch();
   };
 
+  const handleFromInputChange = value => {
```

Comment:
why you dont use  history.replace [(we have on halper)](https://github.com/Bnei-Baruch/kmedia-mdb/blob/d8f2e02d94de1a4b73f8a2f377cc39884016d2f2/src/helpers/url.js#L103)

Reply:
No strong reason, just copied the older code that how it was before.
What should we use?